packages/rollup-full-node/src/app/handler.ts

```diff
@@ -445,16 +445,14 @@ export class DefaultWeb3Handler implements Web3Handler, FullnodeHandler {
     // Generate the calldata which we'll use to call our internal execution manager
     // First pull out the `to` field (we just need to check if it's null & if so set ovmTo to the zero address as that's how we deploy contracts)
     const ovmTo = ovmTx.to === null ? ZERO_ADDRESS : ovmTx.to
+    // TODO: Verify that the nonce on the transaction is correct
     // Construct the raw transaction calldata
-    const internalCalldata = this.generateEOACallCalldata(
+    const internalCalldata = this.generateUnsignedCallCalldata(
```

Comment:
👍 

Would change to `getCallData(...)` and `getTransactionData(...)` to better indicate how these two are used in the grand scheme of things.